shopping_mall/backend/app/routers/chatbot.py

```
@@ -366,6 +382,23 @@ def close_session(
+            from app.models.exchange_request import ExchangeRequest
+            action = json.loads(session.pending_action)
+            if action.get("type") == "exchange_request":
+                exchange = db.query(ExchangeRequest).filter(
+                    ExchangeRequest.id == action["exchange_request_id"],
+                    ExchangeRequest.status == "pending_confirm",
+                ).first()
+                if exchange:
+                    exchange.status = "cancelled"
+                    db.add(exchange)
+        except Exception:
+            pass
```

Ruff (0.15.9)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)